.git-blame-ignore-revs

```diff
@@ -14,3 +14,7 @@
 
 # Format more snapshot / restore relate projects
 559c4e6ef4f9173bbb59043bacd0ac36c7281040
+
+# Format aggregations and related code (server and x-pack)
+d71544976608bdb53fa4d29521fb328e1033ee2f
+4079bc91b9d0a4799557cc794ca5ca153d81d26f
```

Comment:
This one is for the 7.x line and isn't `master`'s lineage. Should we remove it from this list? I don't think it hurts anything to have it, but it confused me a bit when I couldn't find it in `git log` right away..